slices/libnotify4.yaml

```diff
@@ -0,0 +1,17 @@
+package: libnotify4
+
+essential:
+  - libnotify4_copyright
+
+slices:
+  copyright:
```

Comment:
Persoanlly, I'd prefer to the `copyright` slice to be the end of the file, because that's how it is done in the other SDFs for clearer visibility.

Reply:
Done!